tests/cases/types/rem.slint

```diff
@@ -0,0 +1,46 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+global SomeGlobal := {
+    property<rem> global_rem: 4rem;
+}
+
+TestCase := Window {
+    default-font-size: 10px;
+    property<length> normal: 1rem;
+    property<length> double: 2rem;
+    property<length> half: 0.5rem;
+    property <rem> four_rem: 4rem;
+    property<length> four: four_rem;
+
+    property<physical-length> phys-pixel-size: 1rem;
+
```

Comment:
This should also test conversion to and from phx and px. As well as comparison with > and <

Reply:
Yes. Done (and tested before and after langtype fix)